lib/definitions.php

```diff
@@ -760,6 +762,107 @@
+define('RT_TYPE_INST', 4);
+define('RT_TYPE_MOD', 5);
+define('RT_TYPE_START', 6);
+define('RT_TYPE_STOP', 7);
+define('RT_TYPE_REMOVE', 8);
+define('RT_TYPE_OTHER', 9);
+
+
+$RT_TYPE_TYPES = array(
+    RT_TYPE_OFFER => array(
+        'label' => trans('Offer'),
+        'class' => 'lms-ui-rt-ticket-type-offer',
+        'name' => 'RT_TYPE_OFFER'
+    ),
+    RT_TYPE_DOCS => array(
+        'label' => trans('Documents'),
+        'class' => 'lms-ui-rt-ticket-type-docs',
+        'name' => 'RT_TYPE_DOCS'
+        ),
+    RT_TYPE_FAULT => array(
+        'label' => trans('Fault'),
+        'class' => 'lms-ui-rt-ticket-type-fault',
+        'name' => 'RT_TYPE_FAULT'
+        ),
+    RT_TYPE_INST => array(
+        'label' => trans('Instalation'),
+        'class' => 'lms-ui-rt-ticket-type-inst',
+        'name' => 'RT_TYPE_INST'
+        ),
+    RT_TYPE_MOD => array(
+        'label' => trans('Modification'),
+        'class' => 'lms-ui-rt-ticket-type-mod',
+        'name' => 'RT_TYPE_MOD'
+        ),
+    RT_TYPE_START => array(
+        'label' => trans('Start service'),
+        'class' => 'lms-ui-rt-ticket-type-start',
+        'name' => 'RT_TYPE_START'
+        ),
+    RT_TYPE_STOP => array(
+        'label' => trans('Hold service'),
+        'class' => 'lms-ui-rt-ticket-type-stop',
+        'name' => 'RT_TYPE_STOP'
+        ),
+    RT_TYPE_REMOVE => array(
+        'label' => trans('Deinstalation'),
+        'class' => 'lms-ui-rt-ticket-type-remove',
+        'name' => 'RT_TYPE_REMOVE'
+        ),
+    RT_TYPE_OTHER => array(
+	'label' => trans('Other'),
+        'class' => 'lms-ui-rt-ticket-type-other',
+	'name' => 'RT_TYPE_OTHER'
+	),
+    );
+
+//Request Tracker Service Types
+define('RT_SERVICE_INTERNET', 1);
+define('RT_SERVICE_TV', 2);
+define('RT_SERVICE_TEL', 3);
+define('RT_SERVICE_CCTV', 4);
+define('RT_SERVICE_IT', 5);
+define('RT_SERVICE_OTHER', 10);
+
+$RT_SERVICE_TYPES = array(
+    RT_SERVICE_INTERNET => array(
+        'label' => trans('Internet'),
+        'class' => 'lms-ui-rt-ticket-service-internet',
+        'name' => 'RT_SERVICE_INTERNET'
+        ),
+    RT_SERVICE_TV => array(
+        'label' => trans('Television'),
+        'class' => 'lms-ui-rt-ticket-service-tv',
+        'name' => 'RT_SERVICE_TV'
+        ),
+    RT_SERVICE_TEL => array(
+        'label' => trans('Phone'),
+        'class' => 'lms-ui-rt-ticket-service-tel',
+        'name' => 'RT_SERVICE_TEL'
+        ),
+    RT_SERVICE_CCTV => array(
+        'label' => trans('Video surveillance'),
+        'class' => 'lms-ui-rt-ticket-service-cctv',
+        'name' => 'RT_SERVICE_CCTV'
+        ),
+    RT_SERVICE_IT => array(
+        'label' => trans('IT Services'),
+        'class' => 'lms-ui-rt-ticket-service-it',
+        'name' => 'RT_SERVICE_IT'
+        ),
+    RT_SERVICE_OTHER => array(
+        'label' => trans('<!rt>Other'),
+        'class' => 'lms-ui-rt-ticket-service-other',
+        'name' => 'RT_SERVICE_OTHER'
+        ),
+    );
+
```

Comment:
Czy typu usługi dla zgłoszenia nie lepiej powiązań z globalnymi typami usług już istniejącymi?

Reply:
Gdzie one są?

Reply:
Z miłą chęcią.

Reply:
https://github.com/lmsgit/lms/blob/master/lib/definitions.php#L415-L421

Reply:
To widzę tutaj trochę niescisłość.

Proponuję:
- utworzyć $SERVICE_TYPES i używać tego typu w Taryfach jak również w zgłoszeniach,
- połączyć proponowane przezemnie $RT_TYPE oraz $EVENTTYPE

@chilek ?

Reply:
> utworzyć $SERVICE_TYPES i używać tego typu w Taryfach jak również w zgłoszeniach,

Myślę, że to dobry pomysł.

Reply:
Utworzyłem SERVICE_TYPE i zmieniłem w moim commicie nazwy zmiennych.

Reply:
Widzę, że nadal jest oddzielne $RT_SERVICE_TYPES...

Reply:
Zapomniałem wrzucić commita. Poszło.

Reply:
@chilek zapomniałem wrzucić commita. Poszło.